app/waf/src/controllers/adc.controller.ts

```diff
@@ -579,10 +580,20 @@ export class AdcController extends BaseController {
       .get(WafBindingKeys.KeyNetworkDriver)
       .then(async networkHelper => {
         for (let k of Object.keys(adc.networks)) {
+          if (!adc.management) {
```

Comment:
How about to move L583-590 outside the `for`. 
And replace it with `merge(adc.management, {networks: {}});`.